pages/help.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import type { NextPage } from 'next'
+import Layout from '../components/layout/Layout'
+import CSVTable from '../components/CSVTable'
+import { ThemeProvider, Typography, Divider } from '@mui/material'
+import { StyledPageContainer } from '../styles/common'
+import { TextContainer } from '../pageStyles/home.styles'
+import { theme } from '../styles/theme'
+
+const Help: NextPage = () => {
+  return (
+        <Layout>
+        <ThemeProvider theme={theme}>
+          <StyledPageContainer>
+            <Typography variant="h3">Help</Typography>
+            <Typography variant="body1">
+            <br />
+              Use this emailsender to quickly send the same email to multiple recipients at once
+              </Typography>
+            <TextContainer>
+              <Typography variant="h5">Steps: </Typography>
+              <ul>
+                <li>
+                  <Typography variant="body1">
+                      Enter the message inside the email. Leave the recipient name in the form of {'$ {name}'} with no spaces
+                      in between. Leave the company name in the form of <br /> {'$ {company}'} with no spaces
```

Comment:
Depending on the type of email, we may not always have the name or company fields. For example, if we're sending emails to schools or whatever then it'd make sense to use the variable ${school} instead. 

So, in our instructions let's generalize it so that people understand that they can use any field they want. The only required fields are the email and subject field so you can emphasize they need those but the other ones are up to them based on what their email is about.